index.js

```diff
@@ -88,7 +88,9 @@ Router.prototype._on = function _on (method, path, handler, store) {
         isRegex = isRegex || path[i] === '('
         if (isRegex) {
           i = path.indexOf(')', i) + 1
-          break
+          if (i) break
+
+          throw new TypeError('Invalid parameter regexp expression')
```

Comment:
Can you also add the path to the error message?
It will be easier to fix.

Reply:
Done